### DISCUSSION TITLE
Iverter Setup: Gruppieren oder Phasen zuordnen

### DISCUSSION THREAD

**Discussion by Parateam** [tbnobody/OpenDTU#420] [Ideas]
Hallo, ich hätte da eine sinnvolle Idee:

Die Inverter sollten im Setup bei der Seriennummer einer Gruppe zugeordet werden können. Mind. 3 für z.B. die Phasen.

Diese Gruppe sollte dann in der JSON Ausgabe unter total summiert werden.

Hintergrund, ich habe 3 Orte an denen Modulfelder sind. Diese speisen auf 3 unterschiedlichen Phasen ein. Derzeit frage ich alle Inverter einzeln mit Nodered ab, summiere sie nach Phasen auf,  addiere sie mit dem 3-Phasen Plenticore, schreibe sie in ein virtuelles EM24 und habe das in Victron als PV Meter. Läuft gut, könnte mit openDTU und Gruppen vereinfacht werden.

**Comment by Sprinterfreak** [comment]
Das wird eventuell bald möglich sein. Bei dem MQTT Topic Struktur-Update wird es für Strings und Phasen jeweils Freitext-Namensfelder geben. Das lässt dann die Möglichkeit offen, den Einspeisephasen z.B. Sicherungskreis-Bezeichnungen zu geben wie z.B. 1F20 oder Phasenbezeichnungen wie z.B. L1. Aus diesen Informationen lassen sich dann in der zentralen Infrastruktur logische Gruppen bilden.

**Comment by Sprinterfreak** [comment]
See here https://github.com/tbnobody/OpenDTU/discussions/317
